README.md

```diff
@@ -33,6 +33,20 @@ You can install Qt Creator with the ROS corresponding plugin via the [snap store
 sudo snap install qtcreator-ros --classic
 ```
 
+### Debian Qt Creator and Plugin Installation
+
+Qt Creator can be installed via debian officially from [Qt](https://download.qt.io/official_releases/qtcreator). The debian is located in the `cpack_experimental` folder found under each version. Below is an example showing the steps to install both Qt Creator and Plugin. After going through the steps below you must log out and back in so the profile changes are active. Afterwards your system should be able to locate Qt Creator and allow you to add it to your favorits.
+
+```bash
+wget https://download.qt.io/official_releases/qtcreator/14.0/14.0.2/cpack_experimental/qtcreator-opensource-linux-x86_64-14.0.2.deb
+sudo dpkg -i qtcreator-opensource-linux-x86_64-14.0.2.deb 
+echo 'export PATH=$PATH:/opt/qt-creator/bin' >> ~/.bashrc 
+echo 'export PATH=$PATH:/opt/qt-creator/bin' >> ~/.profile 
+echo 'export XDG_DATA_DIRS=$XDG_DATA_DIRS:/opt/qt-creator/share/' >> ~/.profile 
+wget https://github.com/ros-industrial/ros_qtc_plugin/releases/download/14.2/ROSProjectManager-14.2-Linux-x86_64.deb 
+sudo dpkg -i ROSProjectManager-14.2-Linux-x86_64.deb
```

Comment:
> The Debian package has a .dekstop file, so you should be able to launch it from the desktop without setting 

This was not the case for me because it installs into the opt directory with is not searched requiring the changes to `.bashrc` and `.profile`. Not sure if this is just my machine but it is a new machine as of a week ago with a fresh install of Ubuntu 22.04"

Reply:
You are right. The Debian package installs a `.desktop` file to `/opt/qt-creator/share/applications/org.qt-project.qtcreator.desktop` but it is using `Exec=qtcreator %F` and not the absolute path. And it also does not look like desktops are searching for `.desktop` files there. E.g. the office Google Chrome package installs the launcher to `/usr/share/applications/google-chrome.desktop`.

Maybe we should create our own launcher at `/usr/share/applications/qtcreator-ros.desktop` with the absolute path to `/opt/qt-creator/bin/qtcreator`.

Reply:
Creating our own sounds good. This `echo 'export XDG_DATA_DIRS=$XDG_DATA_DIRS:/opt/qt-creator/share/' >> ~/.profile` was required for it to show the application icon.